tools/setup_helpers/ib_detect.py

```diff
@@ -0,0 +1,76 @@
+import os
+import subprocess
+import re
+
+
+WITH_IB_DEVICES = False
+IB_DEVINFO_CMD = "ibv_devinfo"
+
+
+def get_command_path(command):
```

Comment:
Do you really have to resolve the path here? I’m pretty sure that subprocess will correctly take PATH into account. Was it failing for you without it?

Reply:
subprocess will take PATH into account.  The main purpose is to detect if the binary exists. I feel it's cleaner to do it this way.